File: springboot/src/main/java/com/edufelip/meer/security/JwtProperties.java
package com.edufelip.meer.security;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "security.jwt")
public class JwtProperties {
    private String secret = "change-me-please-change-me-please-change-me-123456789012345678901234";
    private long accessTtlMinutes = 60;
    private long refreshTtlDays = 7;

    public String getSecret() { return secret; }
    public void setSecret(String secret) { this.secret = secret; }
    public long getAccessTtlMinutes() { return accessTtlMinutes; }
    public void setAccessTtlMinutes(long accessTtlMinutes) { this.accessTtlMinutes = accessTtlMinutes; }
    public long getRefreshTtlDays() { return refreshTtlDays; }
    public void setRefreshTtlDays(long refreshTtlDays) { this.refreshTtlDays = refreshTtlDays; }
}

File: springboot/src/main/kotlin/com/edufelip/meer/security/JwtProperties.kt
package com.edufelip.meer.security

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "security.jwt")
data class JwtProperties(
    val secret: String = "change-me-please-change-me-please-change-me",
    val accessTtlMinutes: Long = 60,
    val refreshTtlDays: Long = 7
)
